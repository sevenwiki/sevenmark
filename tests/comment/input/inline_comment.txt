Text before comment // This is an inline comment
Text after comment.