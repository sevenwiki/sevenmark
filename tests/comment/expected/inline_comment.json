[
  {
    "Text": {
      "location": {
        "start": 0,
        "end": 20
      },
      "content": "Text before comment "
    }
  },
  {
    "Comment": {
      "location": {
        "start": 20,
        "end": 49
      },
      "content": " This is an inline comment"
    }
  },
  {
    "Text": {
      "location": {
        "start": 49,
        "end": 68
      },
      "content": "Text after comment."
    }
  }
]